src/renderer/src/Layout/Header.tsx

```diff
@@ -3,43 +3,119 @@ import Button from "./Button";
 import Searchbar from "@renderer/Compontents/Searchbar";
 import { useState } from "react";
 import AuthModal from "./AuthModal";
-import { AnimatePresence } from "framer-motion";
+import { AnimatePresence, motion } from "framer-motion";
+import { useAtom } from "jotai";
+import { userAtom } from "@renderer/lib/atoms";
+import { BsDoorOpenFill } from "react-icons/bs";
 
 export default function Header(): JSX.Element {
   const [isUserLoggingIn, setIsUserLoggingIn] = useState<boolean>(false);
+  const [isMenuOpen, setIsMenuOpen] = useState<boolean>(false);
+  const [user, setUser] = useAtom(userAtom);
 
   const menuLinkClass = (isActive: boolean): string => {
     return `text-2xl roboto uppercase transition-all ${
-      isActive ? "text-gray-100" : "text-gray-500"
+      isActive ? "text-emerald-500" : "text-gray-500"
     }`;
   };
 
   function handleLoginModalOpen(): void {
     setIsUserLoggingIn(true);
+    setIsMenuOpen(false);
   }
 
   function handleLoginModalClose(): void {
     setIsUserLoggingIn(false);
   }
 
+  function handleMenuStateChange(): void {
+    setIsMenuOpen((prev: boolean) => !prev);
+  }
+
+  function logoutHandler(): void {
```

Comment:
Please create a method `logout` in `lib/auth.ts` for clearing localStorage items.